go-controller/pkg/node/controllers/egressip/egressip.go

```diff
@@ -0,0 +1,1130 @@
+		if err != nil {
+			return fmt.Errorf("unable to get route list using filter (%s): %v", filter.String(), err)
+		}
+		for _, existingRoute := range existingRoutes {
+			route := routemanager.ConvertNetlinkRouteToRoute(existingRoute)
+			routeStr := route.String()
+			assignedIPRoutes.Insert(routeStr)
+			assignedIPRouteStrToRoutes[routeStr] = routemanager.RoutesPerLink{Link: link,
+				Routes: []routemanager.Route{route}}
+		}
+	}
+	filter, mask := filterRuleByPriority(rulePriority)
+	existingRules, err := util.GetNetLinkOps().RuleListFiltered(netlink.FAMILY_ALL, filter, mask)
+	if err != nil {
+		return fmt.Errorf("failed to list IP rules: %v", err)
+	}
+	for _, existingRule := range existingRules {
+		ruleStr := existingRule.String()
+		assignedIPRules.Insert(ruleStr)
+		assignedIPRulesStrToRules[ruleStr] = existingRule
+	}
+	if c.v4 {
+		ipTableV4Rules, err := c.iptablesManager.GetIPv4ChainRuleArgs(utiliptables.TableNAT, chainName)
+		if err != nil {
+			return fmt.Errorf("failed to list IPTable IPv4 rules: %v", err)
+		}
+		for _, rule := range ipTableV4Rules {
+			ruleStr := strings.Join(rule.Args, " ")
+			assignedIPTableV4Rules.Insert(ruleStr)
+			assignedIPTablesV4StrToRules[ruleStr] = rule
+		}
+	}
+	if c.v6 {
+		ipTableV6Rules, err := c.iptablesManager.GetIPv6ChainRuleArgs(utiliptables.TableNAT, chainName)
+		if err != nil {
+			return fmt.Errorf("failed to list IPTable IPv4 rules: %v", err)
+		}
+		for _, rule := range ipTableV6Rules {
+			ruleStr := strings.Join(rule.Args, " ")
+			assignedIPTableV6Rules.Insert(ruleStr)
+			assignedIPTablesV6StrToRules[ruleStr] = rule
+		}
+	}
+	expectedAddrs := sets.New[string]()
+	expectedIPRoutes := sets.New[string]()
+	expectedIPRules := sets.New[string]()
+	expectedIPTableV4Rules := sets.New[string]()
+	expectedIPTableV6Rules := sets.New[string]()
+	egressIPs, err := c.getAllEIPs()
+	if err != nil {
+		return err
+	}
+	node, err := c.nodeLister.Get(c.nodeName)
+	if err != nil {
+		return err
+	}
+	for _, egressIP := range egressIPs {
+		if len(egressIP.Status.Items) == 0 {
+			continue
+		}
+		for _, status := range egressIP.Status.Items {
+			if isValid := isEIPStatusItemValid(status, c.nodeName); !isValid {
+				continue
+			}
+			eIPNet, err := getFullMaskIPNet(status.EgressIP)
+			if err != nil {
+				return err
+			}
+			isOVNManaged, err := util.IsOVNManagedNetwork(node, eIPNet.IP)
+			if err != nil {
+				return fmt.Errorf("failed to determine if EgressIP %s IP %s is OVN managed: %v", egressIP.Name,
+					status.EgressIP, err)
+			}
+			if isOVNManaged {
+				continue
+			}
+			isV6 := eIPNet.IP.To4() == nil
+			found, link, err := findLinkToHostIP(eIPNet.IP, c.v4, c.v6)
+			if err != nil {
+				return fmt.Errorf("failed to find a network to host EgressIP %s IP %s: %v", egressIP.Name,
+					eIPNet.IP.String(), err)
+			}
+			if !found {
+				return fmt.Errorf("expected to find network interface to assign EgressIP %s IP %s", egressIP.Name,
+					status.EgressIP)
+			}
```

Comment:
If we can't find this we should probably just continue. That EIP wouldn't function without it so it is ok to just remove stale stuff about it and let the reconcile later on rebuild as required.

Reply:
not addressed?

Reply:
Done. Not sure how I lost this change. Thanks for verifying.